setup.cfg

```diff
@@ -2,6 +2,7 @@
 name = falcon
 version = attr: falcon.__version__
 description = The ultra-reliable, fast ASGI+WSGI framework for building data plane APIs at scale.
+long_description = file: README.rst
```

Comment:
I think it makes more sense to make this follow https://github.com/falconry/falcon/pull/2277

Reply:
I have not decided yet whether we make the switch to `pyproject.toml` before 4.0 or post-4.0. So I suggest we merge this as a stopgap measure.

Reply:
I think from a tool prospective is the same.
pyproject seems to the suggested way.

I think it would make sense before 4.0, but it's probably not that big of a deal to migrate after

Reply:
We can migrate before, but we need to go through things like `CONTRIBUTING.md`, because the legacy `tag_build` stuff is gone, etc. I haven't assessed all the potential impact/fallout yet.